modules/application-health-about.adoc

```diff
@@ -9,121 +9,165 @@ A probe is a Kubernetes action that periodically performs diagnostics on a
++
+You can use an HTTP probe with applications that return HTTP status codes when completely initialized.
+
+* Container Execution probe. When using a Container Execution probe, the kubelet executes a command inside the container. The probe is successful if the probe exits with
+a `0` status.
+
+* TCP Socket probe. When using a TCP Socket probe, the kubelet attempts to open a socket to the container. The container is only
+considered healthy if the probe can establish a connection. You can use a TCP socket probe with applications that do not start listening until
+initialization is complete.
+--
++
+--
+Probes have a number of fields that you can use to more precisely control the behavior of Liveness and Readiness checks:
+
+* `initialDelaySeconds`: The time in seconds after the container has started before the probe can be scheduled. The defaults is `0`.
+* `periodSeconds`: The time in seconds that after which the probe is performed. The default is `10`.
+* `timeoutSeconds`: The time in seconds after which the probe times out. The defaults is `1`.
+* `successThreshold`: The minimum number of consecutive successes for the probe to be considered successful after a failure. The value must be `1` for a Liveness check. The default is `1`. 
+* `failureThreshold`: The number of times to retry the probe after a failure. After the specified retries for a Liveness check, the container is restarted. For a Readiness check, the pod is marked `Unready`. For a Startup check, the container is killed and is subject to the pod's `restartPolicy`. The defaults is 3.
+--
++
+[NOTE]
+====
+The `timeoutSeconds` parameter has no effect on the Readiness and Liveness
+checks for Container Execution probes. You can implement a timeout
+inside the probe itself, as {product-title} cannot time out on an exec call into
+the container. One way to implement a timeout in a probe is by using the `exec-timeout` command to run your
+Liveness or Readiness checks, as shown in the examples.
+====
+
+Example health checks::
+The following are samples of different health checks as they would appear in an object specification.  
++
+The following is an example of a Readiness check with an HTTP probe:
++
+-- 
+.Sample Readiness check
+[source,yaml]
+----
+...
+spec:
+  containers:
+  - name: readiness <1>
+    args:
+    image: k8s.gcr.io/goproxy:0.1 <2>
+    readinessProbe: <3>
+      exec: <4>
+        command: <5>
+        - cat
+        - /tmp/healthy
+...
+----
+
+<1> The container name.
+<2> The image to use for the readiness probe.
+<3> The readiness heath check.
+<4> The type of Readiness probe, here an Container Execution probe.
+<5> Specifies the commands to execute on the container.
+--
++
+The following is an example of a Startup check and Liveness check with an HTTP probe:
++
+--
+.Sample Liveness check
```

Comment:
It's better to be "Sample Liveness check and Startup check" for this sample also include a Startup proble